reflex-gadt-api.cabal

```diff
@@ -54,6 +54,11 @@ executable readme
     , reflex-gadt-api
     , text
     , time
+    -- We don't actually need this dependency, but omitting it can cause issues
+    -- when an older dependent-sum is used (from before the some package was
+    -- split out). Since this is already a transitive dependency, this seems
+    -- like the simplest solution.
+    , dependent-sum       >=0.7    && <0.8
```

Comment:
Perhaps there is a better way to prevent this situation from happening. While this fixes our CI, users of the library could fall into the same trap with peculiar error messages: https://github.com/reflex-frp/reflex-gadt-api/runs/2833339637

I can only think of moving this dependency into the library itself, but it seems like there must be a better way.

Reply:
Oh damn I also ran into this: #10